wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java

```diff
@@ -1834,7 +1834,7 @@ else if (timerTickCount < 120 && timerTickCount % 30 == 0)
             }
             catch (Exception e)
             {
-                throw new Exception("Error attempting to refresh WU " + wu.getWuid());
+                throw new Exception("Error attempting to refresh WU " + wu.getWuid() + "\n" + e.getLocalizedMessage());
```

Comment:
Why nut just pass `e` as the caused by to the new exception?  You'll get the message and the trace of where the error occurred.